test/parallel/test-crypto-scrypt.js

```diff
@@ -169,7 +169,7 @@ for (const options of bad) {
 
 for (const options of toobig) {
   const expected = {
-    message: /error:[^:]+:digital envelope routines:EVP_PBE_scrypt:memory limit exceeded/,
+    message: /error:[^:]+:digital envelope routines:[^:]+:memory limit exceeded/,
```

Comment:
I think this is a reasonable approach seeing there's no `.code` or `.opensslErrorStack` property to differentiate on but I would like for the regular expression to stay specific. Can you change this to `(?:EVP_PBE_scrypt|that_redhat_function)`?

Reply:
No problem, I've added it now. Thanks